client/src/elm/Pages/Item/Update.elm

```diff
@@ -1,68 +1,52 @@
-port module Pages.Item.Update
+module Pages.Item.Update
     exposing
-        ( subscriptions
-        , update
+        ( update
         )
 
-import EveryDictList
-import Item.Decoder exposing (decodeItems)
-import ItemComment.Model exposing (ItemComment)
+import App.Types exposing (BackendUrl)
+import Backend.Entities exposing (ItemId)
+import Backend.Item.Model exposing (Item)
+import Backend.Restful exposing (EntityDictList)
+import ItemComment.Model
 import ItemComment.Update
-import Json.Decode exposing (Value, decodeValue)
-import Pages.Item.Decoder exposing (deocdeItemIdAndComments)
-import Pages.Item.Model exposing (Model, Msg(..))
+import Pages.Item.Model exposing (DelegatedMsg(..), Model, Msg(..))
+import StorageKey exposing (StorageKey)
 
 
-update : Msg -> Model -> ( Model, Cmd Msg )
-update msg model =
+update :
```

Comment:
@rgrempel I've wired and adapted some of the concepts based on having a single source of truth data.

The `DelegatedMsg` is bubbled all the way to App.Update -- where it orchestrates the backend (updated the data locally or sending Cmds)

Diff might be hard to read, but we can discuss 😄

Reply:
> Diff might be hard to read, but we can discuss

Your diffs are certainly gentler than mine sometimes are!